tests/integration/integ_test_base.py

```diff
@@ -202,11 +202,13 @@ def setUp(self):
         # startup script
         with open(self.tmp_dir + '/output.txt', 'w') as outfile:
             if platform.system() == 'Windows':
+                self.py = 'python'
                 self.process = subprocess.Popen(
                     ['startup.cmd', self.config_file_name],
                     stdout=outfile,
                     stderr=outfile)
             else:
+                self.py = 'python3'

```

Comment:
I still strongly advise against doing this type of aliasing; this is really only going to confuse people

Reply:
I agree we need better way of finding out Python executable name on a platform. My suggestion is @sbabayan opens issue for doing so and we investigate (or search for how others solve the problem). IMO no need to delay these changes for the imperfection we see here.

Reply:
I will open an issue to track this work